src/ServicePulse.Host/app/css/particular.css

```diff
@@ -769,7 +764,7 @@ p.endpoint-metadata {
     padding-bottom: 0;
 }
 
-.events .box:hover { 
+.events .box:hover, { 
```

Comment:
@WojcikMike I'll fix this once you're done with your changes. Or you can remove the stray comma yourself